recipes/leftbar/general_row/general_row.vue

```diff
@@ -0,0 +1,173 @@
+        type="bulletin"
+        data-qa="leftbar-general-row-unread-badge"
+      >
+        {{ unreadCount }}
+      </dt-badge>
+    </div>
+  </li>
+</template>
+
+<script>
+import { LEFTBAR_GENERAL_ROW_TYPES, LEFTBAR_GENERAL_ROW_CONTACT_CENTER_COLORS } from './general_row_constants.js';
+import { DtBadge } from '@/components/badge';
+import DtEmojiTextWrapper from '@/components/emoji_text_wrapper/emoji_text_wrapper.vue';
+import DtRecipeLeftbarGeneralRowIcon from './leftbar_general_row_icon.vue';
+
+export default {
+  name: 'DtRecipeGeneralRow',
+
+  components: {
+    DtEmojiTextWrapper,
+    DtBadge,
+    DtRecipeLeftbarGeneralRowIcon,
+  },
+
+  props: {
+    /**
+     * Determines the icon to show.
+     * If type is contact center, the color prop must be provided and will determine the color of the icon
+     */
+    type: {
+      type: String,
+      default: 'inbox',
+      validator: (type) => {
+        return Object.values(LEFTBAR_GENERAL_ROW_TYPES).includes(type);
+      },
+    },
+
+    /**
+     * Text displayed next to the icon
+     */
+    description: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Determines the color of the contact center icon
+     */
+    color: {
+      type: String,
+      default: null,
+      validator: (color) => {
+        return Object.keys(LEFTBAR_GENERAL_ROW_CONTACT_CENTER_COLORS).includes(color);
+      },
+    },
+
+    /**
+     * Number of unread messages
+     */
+    unreadCount: {
+      type: Number,
+      default: null,
+    },
+  },
+
+  computed: {
+    leftbarGeneralRowClasses () {
+      return [
+        'leftbar-general-row',
+      ];
+    },
+
+    descriptionClasses () {
+      return [
+        'leftbar-general-row__description',
+        'd-mr-auto',
+        'd-truncate',
+        {
+          'd-fw-bold': this.unreadCount > 0,
+        },
+      ];
+    },
+
+    getIcon () {
+      return this.type === LEFTBAR_GENERAL_ROW_TYPES.CHANNELS && this.hasUnreadMessages
+        ? 'channel bold'
+        : this.type;
+    },
+
+    hasUnreadMessages () {
+      return this.unreadCount > 0;
+    },
+  },
+
+  watch: {
+    $props: {
+      immediate: true,
+      deep: true,
+      handler () {
+        this.validateProps();
+      },
+    },
+  },
+
+  methods: {
+    validateProps () {
+      if (this.type === LEFTBAR_GENERAL_ROW_TYPES.CONTACT_CENTER &&
+          !Object.keys(LEFTBAR_GENERAL_ROW_CONTACT_CENTER_COLORS).includes(this.color)) {
+        console.error('If type is contact center, color must be one of the following: ' +
+          Object.keys(LEFTBAR_GENERAL_ROW_CONTACT_CENTER_COLORS).join(', '));
+      }
+    },
+  },
+};
+</script>
+
+<style lang="less">
+.leftbar-general-row {
+  padding: var(--su6) var(--su8);
+  height: var(--su24);
+  border-radius: var(--su24);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &:hover {
+    background-color: var(--theme-sidebar-row-color-background-hover);
```

Comment:
@francisrupert do we need to update this variable to `black 200` as it's in the design?